Cargo.toml

```diff
@@ -23,10 +23,13 @@ simple_http = [ "base64" ]
 simple_tcp = []
 # Basic transport over a raw UnixStream
 simple_uds = []
+# Enable Socks5 Proxy in transport
+proxy = ["socks"]
```

Comment:
Can we have testing of the "proxy" feature added to ci please (in `contrib/test.sh`).

Reply:
Should have seen those one myself.. Done..